tests/constraints.txt

```diff
@@ -1,10 +1,9 @@
-importlib-metadata==0.22;python_version < "3.8"
+importlib-metadata==0.22
```

Comment:
Hm, how come we still use this on newer Python versions?

Reply:
We don't but this is a constraint file so importlib-metadata won't be installed unless it's required by build.